index.html

```diff
@@ -1855,73 +1855,73 @@ <h2>Advanced Concepts</h2>
+            The roles and information flows for this specification.
+          </figcaption>
+        </figure>
+
+        <ul>
           <li>
-The <a>issuer</a> issues a <a>verifiable credential</a> to the <a>holder</a>.
+An <a>issuer</a> issues a <a>verifiable credential</a> to a <a>holder</a>.
+Issuance must occur before any other actions involving this credential can occur. 
           </li>
           <li>
-The <a>holder</a> might pass on its <a>verifiable credentials</a> to another
-<a>holder</a>.
+A <a>holder</a> might transfer one or more of its <a>verifiable credentials</a>
+to another <a>holder</a>.
           </li>
           <li>
-The final <a>holder</a> presents its <a>verifiable credentials</a> to the
-<a>verifier</a> in a <a>verifiable presentation</a>, requesting a supported
-action.
+A <a>holder</a> presents one or more of its <a>verifiable credentials</a> to a
+<a>verifier</a> in a <a>verifiable presentation</a>.
           </li>
           <li>
-The <a>verifier</a> <a>verifies</a> the authenticity of the
+A <a>verifier</a> <a>verifies</a> the authenticity of the
 <a>verifiable presentation</a> and <a>verifiable credentials</a>.
+This should include checking the <a>credential status</a> for revocation
+of <a>verifiable credentials</a>. Verification occurs some time after a
+<a>verifiable presentation</a>.
           </li>
-          <li>
-The <a>verifier</a> <a>verifies</a> that the <a>holder</a> possesses the
-<a>verifiable credentials</a>.
+         <li>
+An <a>issuer</a> <a>revokes</a> a <a>verifiable credential</a>.
           </li>
           <li>
-The <a>verifier</a> decides whether to accept the <a>verifiable credentials</a>
-for the requested action, taking into account its policy, the <a>holder</a>,
-and the contents of the <a>verifiable credentials</a>.
+A <a>holder</a> deletes a <a>verifiable credential</a>.
           </li>
-          <li>
-The <a>verifier</a> either performs the requested action or rejects the
-request.
-          </li>
-        </ol>
+        </ul>
 
         <p class="note">
-The order of the steps above is not fixed, and some steps might be taken
-more than once. Such step-recurrence may be immediate or at any later
-point in the sequence as performed.
+The order of the actions above is not fixed, and some actions might be taken
+more than once. Such action-recurrence may be immediate or at any later
+point.
         </p>
-
+        
+        <p>
+The simplest useful sequence of actions envisioned would be <a>issue</a>, 
+<a>present</a>, and <a>verify</a>.
         <p>
 This specification does not define any protocol for transfering
 <a>verifiable credentials</a> or <a>verifiable presentations</a>, but assuming
 other specifications do specify how they are transferred between entities, then
-this Verifiable Credential Data Model is:
+this Verifiable Credential Data Model is directly applicable.
+        </p>
+        <p>
+This specification does not define an authorization framework or the
```

Comment:
`framework or the`
-> `framework nor the`

Reply:
Agreed.